lib/Jsrt/ChakraCore.h

```diff
@@ -193,16 +248,19 @@ JsInitializeModuleRecord(
     _Outptr_result_maybenull_ JsModuleRecord* moduleRecord);
 
 /// <summary>
-///     Parse the module source
+///     Parse the source for an ES module
 /// </summary>
 /// <remarks>
-/// This is basically ParseModule operation in ES6 spec. It is slightly different in that the ModuleRecord was initialized earlier, and passed in as an argument.
+/// This is basically ParseModule operation in ES6 spec. It is slightly different in that:
+/// a) The ModuleRecord was initialized earlier, and passed in as an argument.
+/// b) This includes a check to see if the module being Parsed is the last module in the
+/// dependency tree. If it is it automatically triggers Module Instantiation.
 /// </remarks>
-/// <param name="requestModule">The ModuleRecord that holds the parse tree of the source code.</param>
+/// <param name="requestModule">The ModuleRecord being parsed.</param>
 /// <param name="sourceContext">A cookie identifying the script that can be used by debuggable script contexts.</param>
 /// <param name="script">The source script to be parsed, but not executed in this code.</param>
-/// <param name="scriptLength">The source length of sourceText. The input might contain embedded null.</param>
-/// <param name="sourceFlag">The type of the source code passed in. It could be UNICODE or utf8 at this time.</param>
+/// <param name="scriptLength">The length of sourceText in bytes. As the input might contain a embedded null.</param>
```

Comment:
Thanks for adding the "in bytes" phrase here. There is a big point of confusion between "byte count" and "character count"

Reply:
I can't credit for this one - I wrote this content for the wiki first before updating the header with it. The note on bytes was suggested by MSLaguana when reviewing my wiki submission.